contracts/MarketOracle.sol

```diff
@@ -6,15 +6,19 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "./MarketSource.sol";
 
 
+interface IOracle {
+    function getAggregatedData() external returns (uint256);
```

Comment:
How about just `getData()`?

At the high level an oracle just provides an interface to data, whether that's aggregated or not.

Reply:
👍